posthog/settings.py

```diff
@@ -63,6 +63,14 @@ def get_list(text: str) -> List[str]:
         ("️WARNING! E2E_TESTING is set to `True`. This is a security vulnerability unless you are running tests.")
     )
 
+# These flags will be force-enabled on the frontend **in addition to** flags from `/decide`
+# The features here are released, but the flags are just not yet removed from the code.
+PERSIST_FEATURE_FLAGS = get_list(os.getenv("PERSIST_FEATURE_FLAGS", "")) or [
```

Comment:
I was thinking we could allow a special value, e.g. `False` that allows setting this as an empty list (essentially bypassing this behavior). Alternatively or in addition to we may want to ignore this if `IS_MULTITENANCY`

Reply:
Setting this env to `"0"` or `"False"` will effectively achieve the same result now, only there will be one unchecked feature flag floating around. I'd say that's good enough to start :)

Reply:
Regarding cloud... I'd actually leave this on. Otherwise we run the risk of having the self-hosted and cloud environments start to differ by too much...